hits.php

```diff
@@ -3,38 +3,24 @@
  * Ordering Element
  *
  * @package     Joomla.Plugin
- * @subpackage  Fabrik.element.ordering
- * @copyright   Copyright (C) 2024 Jlowcode Org - All rights reserved.
+ * @subpackage  Fabrik.element.hits
+ * @copyright   Copyright (C) 2025 Jlowcode Org - All rights reserved.
  * @license     GNU/GPL http://www.gnu.org/copyleft/gpl.html
  */
 
 // No direct access
 defined('_JEXEC') or die('Restricted access');
 
-use Joomla\CMS\Language\Text;
-use Joomla\CMS\MVC\Model\BaseDatabaseModel;
-use Joomla\CMS\Table\Table;
-use Joomla\CMS\Component\ComponentHelper;
-use Joomla\CMS\Layout\LayoutInterface;
-use Joomla\CMS\Filter\InputFilter;
-use Joomla\CMS\Router\Route;
-use Joomla\CMS\Profiler\Profiler;
 use Joomla\CMS\Factory;
-use Joomla\String\StringHelper;
-use Joomla\Registry\Registry;
-use Joomla\Utilities\ArrayHelper;
-use Joomla\CMS\HTML\HTMLHelper;
-use Fabrik\Helpers\Php;
-
 
```

Comment:
Adicione o método abaixo nesta classe para esconder o elemento do formulário e dos detalhes como haviamos combinado.


```suggestion
/**
    * Is the element hidden or not - if not set then return false
    *
    * @return  bool
    */
public function isHidden()
{
    $element = $this->getElement();

    return ($element->hidden == true) ? true : false;
}
```